templates/cli/lib/commands/push.js.twig

```diff
@@ -864,30 +862,20 @@ const pushCollection = async ({ all, yes } = {}) => {
 
         success(`Created ${relationships.length} relationship attributes`);
     }
+
+    process.exit(0);
```

Comment:
Why do we need to add those exists?

Reply:
Because `inquirer js` doesn't support auto-answer, the workaround for enabling this feature sometimes leaves the console hanging. For that, `process.exit(0)` will ensure that the process finishes.